### PR TITLE
Add security settings & enable nested virtualization

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
Add security settings for opening links & enable nested virtualization for the emulator.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,15 +23,23 @@ RUN mkdir -p /config/.config/openbox \
...
 
-CMD ["/init"]
+# Add security settings for opening links
+RUN apk add --no-cache chromium \
+    && mkdir -p /config/.config/chromium \
+    && echo "user_pref('network.protocol-handler.warn-external-default', true);" > /config/.config/chromium/prefs.js
+
+# Enable nested virtualization
+RUN apk add --no-cache qemu qemu-system-x86_64 \
+    && echo "options kvm_intel nested=1" > /etc/modprobe.d/kvm_intel.conf
```
Comment (Siddhant-K-code) [inline Dockerfile:41]: We might need to check first for Intel/AMD, and then define the conf.


Wdyt?

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     && mkdir -p /opt/android-studio \
     && tar -xvzf /tmp/android-studio.tar.gz -C /opt/android-studio --strip-components=1 \
     && rm /tmp/android-studio.tar.gz
 
-RUN mkdir -p /defaults
-# Set up autostart for Android Studio
-RUN echo "/opt/android-studio/bin/studio.sh" > /defaults/autostart
-RUN mkdir -p /config
+RUN mkdir -p /defaults /config \
+    && echo "/opt/android-studio/bin/studio.sh" > /defaults/autostart
```
Comment (tyvsmith) [inline Dockerfile:15]: Seems like a reasonable cleanup. I was thinking longer term it probably makes sense to move all the generated files in this dockerfile to be checked in and moved where appropriate. Fine for now.

File: Dockerfile (hunk 1 above)
Comment (tyvsmith) [reply Dockerfile:41]: I have an AMD host at home I can give this a test on later. Probably also needs to consider what ARM support would look like, but a good start for the prototype.

File: Dockerfile [hunk 3]
```diff
@@ -23,15 +23,23 @@ RUN mkdir -p /config/.config/openbox \
...
       <decor>no</decor> \
       <fullscreen>yes</fullscreen> \
     </application> \
   </applications> \
 </openbox_config>' > /config/.config/openbox/rc.xml
 
 
-CMD ["/init"]
+# Add security settings for opening links
+RUN apk add --no-cache chromium \
```
Comment (tyvsmith) [inline Dockerfile:35]: I was wanting to explore if it was possible to have the links fire in the user's browser as opposed to hosting chromium within the container. Haven't looked into if that's possible yet.

**Comment by tyvsmith** [comment]
Since I haven't setup any CI or best practices for this repo, I'm approving this from an eyeball review. If there's any issues once I run it later, I'll follow-up with a fix forward or revert if needed.

**Review by tyvsmith — APPROVED** [review]

**Pull request merged**